src/features/shared/components/market-status-led.tsx

```diff
@@ -0,0 +1,223 @@
+        return {
+          status: t('common.after_hours', 'After Hours'),
+          details: t('marketStatus.after_hours_info', {
+            closeTime: marketCloseET,
+            afterHoursClose: afterHoursCloseET,
+            localTime: localServerTime,
+            timeZone: localTimeZone,
+          }),
+        };
+      }
+
+      if (marketStatus.early_hours) {
+        return {
+          status: t('common.pre_market', 'Pre-Market'),
+          details: t('marketStatus.pre_market_info', {
+            openTime: marketOpenET,
+            localTime: localServerTime,
+            timeZone: localTimeZone,
+          }),
+        };
+      }
+
+      return {
+        status: t('common.market_open', 'Market Open'),
+        details: t('marketStatus.regular_hours_info', {
+          openTime: marketOpenET,
+          closeTime: marketCloseET,
+          localTime: localServerTime,
+          timeZone: localTimeZone,
+        }),
+      };
+    }
+
+    if (isWeekend) {
+      return {
+        status: t('common.market_closed', 'Market Closed'),
+        details: t('marketStatus.weekend_closed', {
+          openTime: marketOpenET,
+          localTime: localServerTime,
+          timeZone: localTimeZone,
+        }),
+      };
+    }
+
+    // Check if market opens later today (before 9:30 AM ET)
+    const hours = currentET.getHours();
+    if (hours < 10) {
+      let hoursUntilOpen = 9 - hours;
+      let minutesUntilOpen = 30 - currentET.getMinutes();
+      if (minutesUntilOpen < 0) {
+        hoursUntilOpen -= 1;
+        minutesUntilOpen += 60;
+      }
+      return {
+        status: t('common.market_closed', 'Market Closed'),
+        details: t('marketStatus.opens_in', {
+          openTime: marketOpenET,
+          hours: hoursUntilOpen,
+          hourLabel:
+            hoursUntilOpen === 1 ? t('common.hour') : t('common.hours'),
+          minutes: minutesUntilOpen,
+          localTime: localServerTime,
+          timeZone: localTimeZone,
+        }),
+      };
+    }
+
+    // Market already closed for the day (after 4:00 PM ET)
+    if (hours >= 16) {
+      return {
+        status: t('common.market_closed', 'Market Closed'),
+        details: t('marketStatus.opens_tomorrow', {
+          openTime: marketOpenET,
+          localTime: localServerTime,
+          timeZone: localTimeZone,
+        }),
+      };
+    }
+
+    // Default closed message (fallback case)
+    return {
+      status: t('common.market_closed', 'Market Closed'),
+      details: t('marketStatus.market_closed_default', {
+        localTime: localServerTime,
+        timeZone: localTimeZone,
+      }),
+    };
+  };
+
+  return {
+    isPending,
+    isMarketOpen,
+    marketStatusInfo: getMarketStatusInfo(),
+  };
+}
+
+export function MarketStatusLED() {
+  const { t } = useTranslation();
+  const { isPending, isMarketOpen, marketStatusInfo } = useMarketStatus();
+
+  if (isPending) {
+    return <PendingLED t={t} />;
```

Comment:
can\t find the relevant line now, but when it's pending, don't display the tooltip
<img width="188" height="63" alt="image" src="https://github.com/user-attachments/assets/54d55895-273d-424e-9f20-46ed0cd26680" />